crates/volta-layout/src/v0.rs

```diff
@@ -28,14 +28,14 @@ layout! {
                 "yarn": yarn_image_root_dir {}
                 "packages": package_image_root_dir {}
             }
-            "user": user_toolchain_dir {
-                "bins": user_bin_dir {}
-                "packages": user_package_dir {}
-                "platform.json": user_platform_file;
+            "default": default_toolchain_dir {
```

Comment:
As above, we don't want to change the actual folder that these are put in, so the key here should still be `"user"`. Changing the name of the methods makes sense, however, so those changes should remain.